sessionctx/variable/session.go

```diff
@@ -2389,3 +2396,12 @@ func (s *SessionVars) GetSeekFactor(tbl *model.TableInfo) float64 {
 	}
 	return s.seekFactor
 }
+
+// IsRcCheckTsRetryable checks if the current error is retryable for `RcReadCheckTS` path.
+func (s *SessionVars) IsRcCheckTsRetryable(err error) bool {
+	if err == nil {
+		return false
+	}
+	// The `RCCheckTS` flag of `stmtCtx` is reset d
```

Comment:
```suggestion
	// The `RCCheckTS` flag of `stmtCtx` is set
```
Am I understanding the right way?